slicetag.sh

```diff
@@ -48,7 +48,7 @@ set -e
 command=$1
 shift || :
 
-TAG=$( git rev-list --all | wc -l ) 
+TAG=$( date -u +"%s" ) 
```

Comment:
This will change the date on every rebuild even if the sources have not changed.

I suggest finding a way to extract the date of the most recent commit and using that date as the TAG. This way the date will remain static for every rebuild until a new commit is added.